tests/scripts/git_utils.py

```diff
@@ -23,6 +23,12 @@
 from typing import Dict, Tuple, Any, Optional, List
 
 
+def compress_query(query: str) -> str:
```

Comment:
did we hit a query size limit?

Reply:
no it just makes it easier to debug (i.e. printing the query looks nicer without all the extra whitespace)